Makefile

```diff
@@ -45,7 +45,7 @@ version-check:
         exit 1; \
     fi
 
-build: configcli pkg/apis/kubedirector.bluedata.io/v1alpha1/zz_generated.deepcopy.go version-check | $(build_dir)
+build: configcli pkg/apis/kubedirector.hpe.com/v1beta1/zz_generated.deepcopy.go version-check | $(build_dir)
```

Comment:
why not v1?

Reply:
Conservative. :-) I'd like this to soak for a while with the "beta" tag.

We also have quite a few API-affecting features in the wings, and while we should be able to do this in a backwards-compatible manner, if we're really going to churn the API for a while yet then I'd rather bump the version v1beta1->v1beta2->v1beta3 etc. rather than v1->v2->v3.